.github/workflows/ci.yml

```diff
@@ -41,123 +36,122 @@ jobs:
         if: contains(matrix.os, 'ubuntu')
 
       - name: install stable toolchain
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@v1
         with:
           toolchain: 1.68
           components: clippy
-          profile: minimal
-          override: true
 
       - name: restore cache
-        uses: Swatinem/rust-cache@v1
+        uses: Swatinem/rust-cache@v2
 
       # Clippy packages in deeper-to-higher dependency order
       - name: cargo clippy glazier
-        uses: actions-rs/cargo@v1
-        with:
-          command: clippy
-          args: --manifest-path=Cargo.toml --all-targets --features=x11 --no-default-features -- -D warnings
+        run: cargo clippy --manifest-path=Cargo.toml --all-targets --features=x11 --no-default-features -- -D warnings
 
       # Test packages in deeper-to-higher dependency order
       - name: cargo test glazier
-        uses: actions-rs/cargo@v1
-        with:
-          command: test
-          args: --manifest-path=Cargo.toml --no-default-features --features=x11
+        run: cargo test --manifest-path=Cargo.toml --no-default-features --features=x11
 
   # we test the wayland backend as a separate job
   test-stable-wayland:
     runs-on: ubuntu-latest
     name: cargo clippy+test (wayland)
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - name: install wayland
         run: |
           sudo apt update
           sudo apt install libwayland-dev libpango1.0-dev libxkbcommon-dev
 
       - name: install stable toolchain
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@v1
         with:
           toolchain: 1.68
           components: clippy
-          profile: minimal
-          override: true
 
       - name: restore cache
-        uses: Swatinem/rust-cache@v1
+        uses: Swatinem/rust-cache@v2
 
       - name: cargo clippy glazier
-        uses: actions-rs/cargo@v1
-        with:
-          command: clippy
-          args: --manifest-path=Cargo.toml --all-targets --features=wayland --no-default-features -- -D warnings
+        run: cargo clippy --manifest-path=Cargo.toml --all-targets --features=wayland --no-default-features -- -D warnings
 
       - name: cargo test glazier
-        uses: actions-rs/cargo@v1
+        run: cargo test --manifest-path=Cargo.toml --features wayland --no-default-features
+
+  # we test the accesskit feature as a separate job
+  test-stable-accesskit:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macOS-latest, windows-2019]
```

Comment:
We probably should make sure that we build on linux as well

Reply:
if I do that, I can just fold the extra build tasks into the existing `build-stable` job and reduce some of the work involved.